<?xml version="1.0"?>
<launch>
	<arg name="type" default="1"/>
	<arg name="train_model" default="true"/>
	<arg name="batch_size" default="32"/>
	<arg name="learning_rate" default="1e-3"/>
	<arg name="error_threshold" default="0.999"/>
	<arg name="turn" default="0"/>
	<arg name="exp_name" default="reddoor" />

<!-- 	<arg name="teleop" default="joy" /> -->
<!-- 	<include file="$(find segway_sim)/launch/rviz.launch" if="$(eval viz == 'on')"/> -->
<!-- 	<group ns="uncertainty_estimator"> -->
		<node name="image_receiver" pkg="uncertainty_estimator" type="image_receiver" output="screen">
<!-- 			<param name="dt" value="0.001" /> -->
<!-- 			<rosparam param="IC">[0.5, 4.5, 3.14, 0.0, 0.0, 0.138322, 0.0]</rosparam> -->
				<param name="experiment_type" type="int" value="$(arg type)"/>
				<param name="batch_size" type="int" value="$(arg batch_size)"/>
				<param name="train_model" type="bool" value="$(arg train_model)" />
				<param name="learning_rate" type="double" value="$(arg learning_rate)"/>
				<param name="error_threshold" type="double" value="$(arg error_threshold)"/>
				<param name="turn" type="int" value="$(arg turn)"/>
		</node>
 	<node pkg="rosbag" type="record" name="record" output="screen"
	args="record -a -o /home/drew/Aeronvironment/catkin_ws_rkc/src/uncertainty_estimator/newbags/$(arg exp_name)_type($(arg type))_train($(arg train_model))_b($(arg batch_size))" />

<!--		  args="record -a -o /home/drew/Aeronvironment/catkin_ws_rkc/src/uncertainty_estimator/bags/quad_barrier_robust"/>-->


<!--							  args="record -a -o /home/drew/Aeronvironment/catkin_ws_rkc/src/uncertainty_estimator/bags/pretest"/>-->







<!--				  args="record -a -o /home/drew/Aeronvironment/catkin_ws_rkc/src/uncertainty_estimator/bags/quad_barrier_robust"/>-->


</launch>



